ludwig/backend/base.py

```diff
@@ -129,3 +134,57 @@ def __init__(self, **kwargs):
 
     def initialize(self):
         pass
+
+    def export_predictions(
+            self,
+            postprocessed_output,
+            output_directory
+    ):
+        # LocalBackend save predictions in csv format
+        # for each column in the postprocessed_output dataframe save as a csv
+
+        # setup to parse column names to form csv files
+        # column names are one of the following forms
+        #   <feature_name>_predictions
+        #   <feature_name>_probabilities
+        #   <feature_name>_probability
+        #   <feature_name>_probabilities_<token_text>
+        pattern = re.compile(
```

Comment:
Would this function work for this? It should result in a dict with the correct format:

https://github.com/ludwig-ai/ludwig/blob/master/ludwig/data/postprocessing.py#L87